src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3213,6 +3213,58 @@ private BoundExpression BindArgumentExpression(BindingDiagnosticBag diagnostics,
         }
 
 #nullable enable
+        private void CheckRefArguments<TMember>(
```

Comment:
It looks like all calls are combined with ```CoerceArguments``` calls. Why not incorporate this check into ```CoerceArguments```? We can change it to take ```AnalyzedArguments```, if that is necessary.